frontend/src/views/Project/SshCaPage/components/SshCertificateModal.tsx

```diff
@@ -0,0 +1,369 @@
+import { useEffect, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+import { createNotification } from "@app/components/notifications";
+import {
+  Button,
+  FormControl,
+  Input,
+  Modal,
+  ModalContent,
+  Select,
+  SelectItem
+} from "@app/components/v2";
+import { useWorkspace } from "@app/context";
+import {
+  SshCertTemplateStatus,
+  useGetSshCertTemplate,
+  useIssueSshCreds,
+  useListWorkspaceSshCertificateTemplates,
+  useSignSshKey
+} from "@app/hooks/api";
+import { certKeyAlgorithms } from "@app/hooks/api/certificates/constants";
+import { CertKeyAlgorithm } from "@app/hooks/api/certificates/enums";
+import { SshCertType } from "@app/hooks/api/ssh-ca/constants";
+import { UsePopUpState } from "@app/hooks/usePopUp";
+
+import { SshCertificateContent } from "./SshCertificateContent";
+
+const schema = z.object({
+  templateId: z.string(),
+  publicKey: z.string().optional(),
+  keyAlgorithm: z.enum([
+    CertKeyAlgorithm.RSA_2048,
+    CertKeyAlgorithm.RSA_4096,
+    CertKeyAlgorithm.ECDSA_P256,
+    CertKeyAlgorithm.ECDSA_P384
+  ]),
+  certType: z.nativeEnum(SshCertType),
+  principals: z.string(),
+  ttl: z.string().optional(),
+  keyId: z.string().optional()
+});
+
+export type FormData = z.infer<typeof schema>;
+
+type Props = {
+  popUp: UsePopUpState<["sshCertificate"]>;
+  handlePopUpToggle: (popUpName: keyof UsePopUpState<["sshCertificate"]>, state?: boolean) => void;
+};
+
+type TSshCertificateDetails = {
+  serialNumber: string;
+  signedKey: string;
+  privateKey?: string;
+  publicKey?: string;
+};
+
+enum SshCertificateOperation {
+  SIGN_SSH_KEY = "sign-ssh-key",
+  ISSUE_SSH_CREDS = "issue-ssh-creds"
+}
+
+export const SshCertificateModal = ({ popUp, handlePopUpToggle }: Props) => {
```

Comment:
In the SSH issuance modal, certificate template is blank and certificate type has no selectable options even when user certificate is enabled for the template 
<img width="1027" alt="image" src="https://github.com/user-attachments/assets/f0c6db82-64a3-4543-89a5-9953aca9ace9">
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/8f56a06e-8c18-467a-b66d-e1f4bbfb4c54">

Reply:
Fixed